x-pack/platform/plugins/private/upgrade_assistant/server/lib/es_deprecations_status/__snapshots__/index.test.ts.snap

```diff
@@ -80,20 +80,39 @@ Object {
     },
     Object {
       "correctiveAction": Object {
+        "excludedActions": Array [],
```

Comment:
Since we've removed the excludeDeprecation logic, we now have more deprecations that match the test assertion.